school_management/tests/test_leave_wizard.py

```diff
@@ -0,0 +1,41 @@
+from datetime import date, timedelta
+
+from odoo.tests.common import TransactionCase
+
+
+class TestLeaveWizard(TransactionCase):
+    def setUp(self):
+        super(TestLeaveWizard, self).setUp()
+
+        self.leave_wiz_record = self.env["teacher.leave.application.wizard"].create(
+            {
+                "date_of_leave_start_wizard": date.today(),
+                "date_of_leave_end_wizard": (date.today() + timedelta(days=7)),
+                "reason_for_leave_wizard": "KHello",
+            }
+        )
+
+    def test_01_compute_total_leave_days(self):
```

Comment:
add proper name for method not test_01_compute_total_leave_days, it should be like test_check_total_leave_days